RockLib.UniversalMemberAccessor.Tests/TypeCreator/Create.cs

```diff
@@ -60,13 +60,13 @@ public static Type Struct(string name, Type[] interfaces, params MemberDefinitio
 
         private static Type Type(
             string name, Type[] interfaces, MemberDefinition[] memberDefinitions,
-            TypeAttributes typeAttributes, Type baseType = null)
+            TypeAttributes typeAttributes, Type? baseType = null)
         {
-            if (name == null) throw new ArgumentNullException("name");
-            if (interfaces == null) throw new ArgumentNullException("interfaces");
-            if (memberDefinitions == null) throw new ArgumentNullException("memberDefinitions");
+            if (name is null) throw new ArgumentNullException(nameof(name));
+            if (interfaces is null) throw new ArgumentNullException(nameof(interfaces));
+            if (memberDefinitions is null) throw new ArgumentNullException(nameof(memberDefinitions));
 
-#if NET40
+#if NET48
```

Comment:
It looks like `AssemblyBuilder.DefineDynamicAssembly` should be available for all our target frameworks: https://docs.microsoft.com/en-us/dotnet/api/system.reflection.emit.assemblybuilder.definedynamicassembly?view=net-6.0#system-reflection-emit-assemblybuilder-definedynamicassembly(system-reflection-assemblyname-system-reflection-emit-assemblybuilderaccess). We may not need the `#if` here.

Reply:
I removed it and everything still works.